script.js

```diff
@@ -0,0 +1,12 @@
+const buttons = document.getElementsByClassName("buttonsBlock");
```

Comment:
it won't work. getElementsByClassName returns an array of elements. Use destruction`[buttons]`